stacks/api/bucket/bucket-data.service.ts

```diff
@@ -2,11 +2,12 @@ import {Injectable} from "@nestjs/common";
 import {
```

Comment:
I think we'll be better if we use `BaseCollection` helper for this service. Would it take too much time to do it?

Reply:
It won't take too much time for this service but the other services like passport, identity has more complicated logics so they might take.